oxidation/libparsec/crates/types/src/lib.rs

```diff
@@ -37,6 +37,7 @@ pub use organization::*;
 pub use pki::*;
 pub use time::*;
 pub use user::*;
+pub use uuid;
```

Comment:
I choose to re-export `uuid` because I need it in the error of `platform_storage` and I wanted to have a simple way to use the same version has `libparsec_types`.

I could also have added `uuid` as a dependency would you prefer that ?

Reply:
In a long term, I would like to use one of our `UUID` struct for this error, I chose raw `uuid`, because I didn't know what to choose

Reply:
That would be nice, for those type are defined as different type.

We could easily convert from one to another `ChunkID => BlockID`.

We could also define `CHunkID` and `BlockID` to be type alias of `EntryID` WDYT ?

Reply:
@touilleMan

Reply:
We've discussed about it yesterday with @touilleMan.

We prefer to keep `ChunkID` & Co as strict different type. 

The need for that re-expose would soon be out-of-date with the specialization of `LocalMiss` (that required `Uuid` currently)